csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/DependencyManager.cs

```diff
@@ -71,7 +71,7 @@ public DependencyManager(string srcDir, ILogger logger)
             this.generatedSources = new();
             var allProjects = allNonBinaryFiles.SelectFileNamesByExtension(".csproj").ToList();
             var allSolutions = allNonBinaryFiles.SelectFileNamesByExtension(".sln").ToList();
-            var dllPaths = allFiles.SelectFileNamesByExtension(".dll").ToHashSet();
+            var dllPaths = allFiles.SelectFileNamesByExtension(".dll").Select<string, AssemblyPath>(x => x).ToHashSet();
```

Comment:
Just an opinion: I find `.Select<string, AssemblyPath>(x => x)` difficult to read, I think `.Select(x => new AssemblyPath(x))` is easier to understand.